cmd/snapctl/task.go

```diff
@@ -135,7 +136,14 @@ func createTaskUsingTaskManifest(ctx *cli.Context) {
 		fmt.Println("Invalid version provided")
 		os.Exit(1)
 	}
-	r := pClient.CreateTask(t.Schedule, t.Workflow, t.Name, t.Deadline, !ctx.IsSet("no-start"))
+
+	// If the number of failure does not specific, default value is 10
+	if t.Failure == 0 {
+		fmt.Println("Not specific the number of failures to disable the task.  Use default value: 10")
+		t.Failure = 10
```

Comment:
Since we reference the default failure number in multiple places, could we make it a const in this package so that if we decide to change it it's just in one place.